.github/workflows/main.yml

```diff
@@ -43,6 +43,13 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+    
+    - name: Install Poetry
+      uses: snok/install-poetry@v1
+      with:
+        virtualenvs-create: true
+        virtualenvs-in-project: true
```

Comment:
There're some errors on [lint stage](https://github.com/drgarcia1986/simple-settings/runs/4652198042?check_suite_focus=true#step:7:8) because of the fact that `isort` is trying to check files inside the virtualenv, can we change this?

Reply:
Changed the Make recipe to look under the simple_settings folder, I think this will do.